examples/_deployed/kitt/kitt.py

```diff
@@ -16,6 +16,11 @@
 
 MAX_IMAGES = 3
 
+NO_IMAGE_MESSAGE_SIP = "I'm sorry, I can't see anything when you're calling from a phone. Try the web demo."
```

Comment:
What about just removing the FunctionContext when using SIP? So we don't need this message and it simplifies the code

Reply:
Yeah makes sense.